node/src/BaseClient.ts

```diff
@@ -2818,6 +2819,46 @@ export class BaseClient {
         return this.createWritePromise(scriptInvocation);
     }
 
+    /**
+     * Returns stream entries matching a given range of IDs.
+     *
+     * See https://valkey.io/commands/xrange for more details.
+     *
+     * @param key - The key of the stream.
+     * @param start - The starting stream ID bound for the range.
+     *     - Use `value` to specify a stream ID.
+     *     - Use `isInclusive: false` to specify an exclusive bounded stream ID.
+     *     - Use `-` to start with the minimum available ID.
+     * @param end - The ending stream ID bound for the range.
+     *     - Use `value` to specify a stream ID.
+     *     - Use `isInclusive: false` to specify an exclusive bounded stream ID.
+     *     - Use `+` to end with the maximum available ID.
+     * @param count - An optional argument specifying the maximum count of stream entries to return.
+     *     If `count` is not provided, all stream entries in the range will be returned.
+     * @returns A mapping of stream IDs to stream entry data, where entry data is a
```

Comment:
Optional: you can copy this from another command which returns same data structure.